### DISCUSSION TITLE
slow build

### DISCUSSION THREAD

**Discussion by Morranto** [dragonflydb/dragonfly#266] [Q&A]
hi!  thanks for your  excellent  project.

but the speed of building dragonflydb  in my ubuntu server is very slow....( very slow  when i run  "  ./helio/blaze.sh -release "    )
i dont know why. Is this normal?

**Comment by romange** [comment]
How long does it take?

**Reply by Morranto** [reply]
hi，it takes  about 15～20minutes

**Reply by romange** [reply]
you mean `./helio/blaze.sh -release` takes 15min or `ninja drafongly` takes 15min?
